index.html

```diff
@@ -8742,6 +8736,10 @@
 							<div class="justifyleft settingsmall" title="Inverts all colors, simple light mode">Inverted Colors </div>
 						   <input type="checkbox" id="invert_colors" style="margin:0px 0 0;">
 						</div>
+						<div class="settinglabel">
+							<div class="justifyleft settingsmall" title="Switches to an aesthetic messenger style UI. Only applicable to Chat Mode & Instruct mode.">Aesthetic Chat UI </div>
+							<input type="checkbox" id="enhanced_chat_ui" style="margin:0px 0 0;">
+						</div>
 					</div>
```

Comment:
Personally I prefer the previous approach of placing this toggle within the dropdown category, because aesthetic mode is inapplicable to adventure and story, and placing the toggle outside implies that it is. People will think it's broken when they toggle it in story mode and nothing changes

Reply:
I think it'd be cool to eventually switch it towards a toggle & (maybe ⚙️) btn that does smth like: 'customize UI from json'.
If/when customization panel is added for the UI, users might wanna store their custom settings and share them with others.
Then 'Kobold-Enhanced-UI' mode could be an option from a dropdown, along with 'Kobold-Classic-UI' mode.

Thought a bit way too much on it without sharing anything and it's embarassing to share xD ..but felt like a cool idea and fairly adjacent to implement along with adding customization options for instruct mode's UI.

Let me know whether you'd like me to move it back and include a separate toggle for it in instruct settings for now or not.

Reply:
I'd be concerned about the maintainability of having too many UIs to be honest, but anyway for now I think following the original aesthetic chat UI toggle placement will probably be best, so yeah I think move it back. 

The themes/customizability can be added later outside of the settings panel - it would be maybe a different panel that only appears within the aesthetic mode. 

Ideally I want to keep the aesthetic UI separate from the regular UI (like how it works now, it's parasitic HTML that hooks onto the main ui for input/output) so that any issues with it won't break the main ui.